### PR TITLE
Cover/cmds/exp/ed cmds - Part 2

### PR DESCRIPTION
**Pull request by ChriMarMe**
Ignore @llogen s commits. I just branched from his branch.

### CODE REVIEW

File: cmds/exp/ed/ed.go [hunk 1]
```diff
@@ -72,63 +85,69 @@ func run(cmd string) (e error) {
...
-		fmt.Fprintf(os.Stderr, "error reading stdin: %v", inScan.Err())
+		return fmt.Errorf("error reading stdin: %v", inScan.Err())
+	}
+	return nil
+}
+
+// Entry point
+func main() {
+	flag.Parse()
+	file := ""
```
Comment (llogen) [inline cmds/exp/ed/ed.go:141]: if you pass flag.Args() to runEd, you can do this switch in runEd to cover more lines.

**Review by llogen — COMMENTED** [review]
Just one thingy.

**Review by llogen — DISMISSED** [review]

File: cmds/exp/ed/ed.go (hunk 1 above)
Comment (ChriMarMe) [reply cmds/exp/ed/ed.go:141]: Yeah true, but some for ed is is not really clear what arguments you should pass. Ron had some issues with this passing flag.Args() in other PRs. especially if there is only one arugment (here file) to pass to the program.

Comment (llogen) [reply cmds/exp/ed/ed.go:141]: Okay

**Pull request closed without merging**